test/kubemark/stop-kubemark.sh

```diff
@@ -45,3 +43,4 @@ rm -rf "${RESOURCE_DIRECTORY}/addons" \
 	"${RESOURCE_DIRECTORY}/kubemark-master-env.sh"  &> /dev/null || true
 
 delete-master-instance-and-resources
+set +x
```

Comment:
If these are intended to stay and not just debugging, please comment

Reply:
That was a temporary change to troubleshoot previous failure.